vcloud.yaml

```diff
@@ -1,11 +1,334 @@
+          format: binary
+        token:
+          type: string
+          description: Unique string used to authenticate and authorize the
+            request.
+          example: "123e4567e89b12d3a456426655440000"
+        url:
+          type: string
+          description: Use as an alternative to the `file` parameter for sending
+            audio in a request. The provided URL must support HTTP `GET` and
+            return a `Content-Length` header when queried.
+          example: "https://username:password@hostname.com/path/to/sample.wav"
+        callback:
+          type: string
+          description: Provide a callback address for V-Cloud to send the
+            results when the transcription process is complete.
+          example: "https://username:password@hostname.com"
+        diarize:
+          type: boolean
+          description: Use `diarize` to identify distinct speakers on mono
+            (single channel) audio and segment detected speech into separate
+            channels, which are identified in JSON output.
+          default: false
+        emotion:
+          type: boolean
+          description: Emotion detection feature that uses a synthesis of
+            acoustic features and word sentiment scores to determine if a
+            given utterance is Mostly Positive, Positive, Neutral, Negative,
+            or Mostly Negative.
+          default: false
+        gender:
+          type: boolean
+          description: Gender detection feature that attempts to determine
+            if the speaker is male or female in each utterance.
+          default: false
+        metadata:
+          type: string
+          description: Use `metadata` to enable the inclusion of user-defined
+            metadata in a top-level object in JSON transcripts.
+        model:
+          type: string
+          description:
+            Specify the [language model](https://docs.vocitec.com/en/language-support.html)
+            to use for transcription.
+          default: eng-us:callcenter
+        music:
+          type: boolean
+          description: Setting `music` to true enables music detection. When
+            music detection is enabled, all utterances pass through an algorithm
+            to be classified as music or speech. Utterances classified as
+            music are assumed to contain noisy audio and are not transcribed.
+          default: false
+        numtrans:
+          type: boolean
+          description: Controls whether or not number words in transcribed
+            text are converted into numeric digits and related conventional
+            formats, including dollar amounts, wall-clock times, percentages,
+            ordinals, and telephone numbers.
+          default: true
+        output:
+          type: string
+          description: Specify the desired output format for transcripts.
+          default: json
+          enum:
+            - json
+            - jsontop
+            - text
```

Comment:
Hint at what each format is (esp. w.r.t. an example) and/or link to some other location that describes in further detail.